opensearchapi/opensearchapi.response.go

```diff
@@ -96,19 +117,41 @@ func (r *Response) Status() string {
 }
 
 // IsError returns true when the response status indicates failure.
-//
 func (r *Response) IsError() bool {
 	return r.StatusCode > 299
 }
 
+// Err returns an error when the response status indicates failures.
+func (r *Response) Err() error {
+	if r.StatusCode > 299 {
```

Comment:
Should we reuse `IsError()` here, and add explicit tests for it?

Reply:
Using IsError() now and also update USER_GUID